src/main/resources/applicationContext-LutungMandrill-all.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I removed the dependency on spring, so I think this could be a nice help for someone who would want to use it (it wouldn't affect any code)